pom.xml

```diff
@@ -102,17 +102,6 @@
     </dependencies>
   </dependencyManagement>
   <profiles>
-    <profile>
-      <id>ice35</id>
-      <activation>
-        <property>
```

Comment:
Does this mean that by default no profile is activated at the moment? Should we activate `ice36` by default?